robot/robot.py

```diff
@@ -38,23 +38,26 @@ def __init__(self, robotd_path=ROBOTD_ADDRESS):
             self.known_gamestates,
         ]
 
+        self._display_boards()
         self._wait_for_power_board()
 
+        print('Waiting for start button.')
+        self.power_board.wait_start()
+
+        self.power_board.power_on()
```

Comment:
This should be done before `wait_start` rather than after, to give boards (e.g. motor board) time to initialise and report themselves ready before the user code starts trying to use them (which may be immediately after `__init__` returns).